### PR TITLE
Lots of small edits

### PR DESCRIPTION
**Pull request by monksharru**
- Removed unnecessary tables of contents
- Removed starting bug from FAQs 🥳
- Removed important information about your operating system from FAQs
- Replaced red text in FAQs with an admonition
- Deleted Minecraft FAQ page because the only thing on it was how to add mods
- Added plugin server menu, proxy server menu and other server menu screenshots
- Added server_types.md to nav
- Made server types page
- Removed server types section from index.md and inserted a link to new server tyoes oage
- Fixed Bukkit link in downloading plugins
- Changed `plugins` to `datapacks` in downloading datapacks
- Deleted the FAQ section and made FAQ its single page
- Maybe some more stuff? idk

### CODE REVIEW

**Comment by monksharru** [comment]
oh boy

File: docs/faq.md [hunk 1]
```diff
@@ -30,17 +12,6 @@ If it gives you an error, you don't have it installed. You can download it [here
...
-
-Go to `File > Options > Advanced` and change the default detection method to `ConsoleReading`.
-
-![Screenshot of the process name requires different format dialog](assets/screenshots/online_detection.png)
-
-
-<a name="unable-to-connect-to-the-server"></a>
 ## Unable to connect to the server.
 
 As the server owner, try connecting with "localhost" (without the "").
```
Comment (vaio2005) [inline docs/faq.md:17]: That works only if the server is hosted on the same machine as the client. That could not be the case.

**Review by vaio2005 — APPROVED** [review]
All good on changes

File: docs/faq.md (hunk 1 above)
Comment (monksharru) [reply docs/faq.md:17]: Sure, editing now.

**Review by vaio2005 — APPROVED** [review]
110/100

**Comment by Fiahblade** [comment]
Added some tweaks, also moved the plugins/mods info to a separate page.

**Pull request merged**